datamol/mol.py

```diff
@@ -1008,11 +1009,12 @@ def to_scaffold_murcko(mol: Mol, make_generic: bool = False):
 
     if make_generic:
         scf = make_scaffold_generic(scf)
+        scf = from_smarts(scf)
```

Comment:
Should add some comment here because, this differ from the rdkit behaviour (return a `to_mol` and not `from_smarts` molecule) 

https://www.rdkit.org/docs/source/rdkit.Chem.Scaffolds.MurckoScaffold.html

Reply:
Hum actually returning `to_mol` would be consistent `make_generic=False` so I think it's best. What do you think?

Reply:
Then `make_scaffold_generic` can just return the molecule as is (same as rdkit) ?
https://github.com/rdkit/rdkit-orig/blob/57058c886a49cc597b0c40641a28697ee3a57aee/rdkit/Chem/Scaffolds/MurckoScaffold.py#L18

Then for practical use, we can then sanitize and convert to smiles/ query mol on downstream function that needs the conversion ?

Reply:
Yup it's likely best that way

Reply:
done